src/main/java/io/tiledb/java/api/Util.java

```diff
@@ -36,4 +36,29 @@ public static int castLongToInt(long num) throws TileDBError {
 
     return (int) num;
   }
+
+  /**
+   * Returns the Datatype of the input attribute
+   *
+   * @param array The TileDB array
+   * @param attr The attribute name
+   * @return The Datatype
+   * @throws TileDBError A TileDBError
+   */
+  public static Datatype getAttributeDatatype(Array array, String attr) throws TileDBError {
```

Comment:
Let's change the second parameter to not call it attr, lets make it more clear this supports attribute or dimensions.